Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:16
```

Comment:
Do you have a node install you could use and then put this as "FROM alpine" or "FROM ubuntu" and then install node in the Dockerfile?

Reply:
Thats what I wrote initially FROM UBUNTU, then I saw looked at the link you sent me and changed it to that. Thought you were heading a different direction using the links FROM node.